wazp/callbacks/dashboard.py

```diff
@@ -0,0 +1,658 @@
+    for f in pl.Path(
+        app_storage["config"]["pose_estimation_results_path"]
+    ).iterdir():
+        if str(f).endswith(".h5"):
+            list_videos_w_pose_results.append(re.sub("DLC.*$", "", f.stem))
+
+    # append status of h5 file per video to table
+    df_metadata[POSE_DATA_STR] = [
+        TRUE_EMOJI if v in list_videos_w_pose_results else FALSE_EMOJI
+        for v in list_videos_w_metadata
+    ]
+
+    # table component
+    return dash_table.DataTable(
+        id="video-data-table",
+        data=df_metadata.to_dict("records"),
+        selected_rows=[],
+        editable=False,
+        row_selectable="multi",
+        fixed_rows={"headers": True},
+        page_size=4,
+        page_action="native",
+        sort_action="native",
+        sort_mode="single",
+        style_header={
+            "backgroundColor": "rgb(210, 210, 210)",
+            "color": "black",
+            "fontWeight": "bold",
+            "textAlign": "left",
+            "fontFamily": "Helvetica",
+        },
+        style_table={
+            "height": "200px",
+            "maxHeight": "200px",
+            "width": "100%",
+            "maxWidth": "100%",
+            # "overflowY": "scroll",
+            # "overflowX": "scroll",
+        },
+        style_cell={  # refers to all cells (the whole table)
+            "textAlign": "left",
+            "padding": 7,
+            "fontFamily": "Helvetica",
+            # "minWidth": 15,
+            # "width": 45,
+            # "maxWidth": 450, #450,
+        },
+        style_header_conditional=[
+            {
+                "if": {"column_id": "File"},
+                # TODO: consider getting file from app_storage
+                "backgroundColor": "rgb(200, 200, 400)",
+            },
+            {
+                "if": {"column_id": POSE_DATA_STR},
+                # TODO: consider getting file from app_storage
+                "textAlign": "center",
+            },
+        ],
+        style_data_conditional=[
+            {
+                "if": {"column_id": POSE_DATA_STR},
+                "textAlign": "center",
+                "minWidth": 200,
+                "width": 200,
+                "maxWidth": 200,
+            },
+            {
+                "if": {
+                    "column_id": app_storage["config"][
+                        "metadata_key_field_str"
+                    ],
+                    "row_index": "odd",
+                },
+                # count starts with 0 (even) at first row with data
+                "backgroundColor": "rgb(220, 220, 420)",
+                # darker blue
+            },
+            {
+                "if": {
+                    "column_id": app_storage["config"][
+                        "metadata_key_field_str"
+                    ],
+                    "row_index": "even",
+                },
+                "backgroundColor": "rgb(235, 235, 255)",
+                # lighter blue
```

Comment:
Did you look into factoring out the css/stylistic stuff? If so is it easy? Straightforward?

Reply:
I think it should be fairly easy but haven't looked into it in detail yet. I added it as a task in #56 for now